packages/expo-file-system/ios/EXFileSystem/EXFileSystem.m

```diff
@@ -675,98 +651,107 @@ - (NSDictionary *)encodingMap
 
 #pragma mark - Internal methods
 
-- (void)_downloadResumableCreateSessionWithUrl:(NSURL *)url withScopedPath:(NSString *)scopedPath withUUID:(NSString *)uuid withOptions:(NSDictionary *)options withResumeData:(NSData * _Nullable)resumeData withResolver:(UMPromiseResolveBlock)resolve withRejecter:(UMPromiseRejectBlock)reject
+- (NSURLSession *)_createSession:(id<NSURLSessionDelegate>)delegate withOptions:(NSDictionary *)options
+{
+  NSNumber *sessionType = options[@"sessionType"] ?: 0;
+  NSDictionary *headers = options[@"headers"];
+  NSURLSessionConfiguration *sessionConfiguration;
+  
+  if (sessionType == 0) {
+    // background session
+    sessionConfiguration = [NSURLSessionConfiguration backgroundSessionConfigurationWithIdentifier:[[NSUUID UUID] UUIDString]];
+  } else {
+    // foreground session
+    sessionConfiguration = [NSURLSessionConfiguration defaultSessionConfiguration];
+  }
+  
+  sessionConfiguration.requestCachePolicy = NSURLRequestReloadIgnoringLocalCacheData;
+  sessionConfiguration.URLCache = nil;
+  if (headers != nil) {
+    sessionConfiguration.HTTPAdditionalHeaders = headers;
+  }
+   
+  return [NSURLSession sessionWithConfiguration:sessionConfiguration
+                                       delegate:delegate
+                                  delegateQueue:[NSOperationQueue mainQueue]];
+}
+
+- (BOOL)_checkIfFileExists:(NSString *)path
+{
+  return [[NSFileManager defaultManager] fileExistsAtPath:path];
+}
```

Comment:
Have you replaces all occurrences of `[[NSFileManager defaultManager] fileExistsAtPath:path]` with this method?